examples/browserify.js

```diff
@@ -1,7 +1,7 @@
 var d3 = require("d3"),
     cloud = require("../");
 
-var fill = d3.scale.category20();
+var fill = d3.scaleOrdinal(d3.schemeCategory20);
```

Comment:
The example was updated for d3 v4 in this pull request: https://github.com/jasondavies/d3-cloud/pull/97